### ISSUE TITLE
Module 3: Operations

### ISSUE THREAD

**Issue by ryandkiley** [castl350/STEP#249]
1. What is Adding and Subtracting
Feedback for wrong answer says "Incorrect. Please try again." Did we create more constructive feedback?

**Comment by ryandkiley** [comment]
What is Adding and Subtracting video.... I know this ship has sailed, but for future development, I believe it should be more and "fewer" not "less"

**Comment by ryandkiley** [comment]
What is Adding and Subtracting video.... result, change, and start unknown discussion seems complicated and the visual shows an addition equation even though earlier in the video we said those symbols wouldn't be discussed.

**Comment by ryandkiley** [comment]
2. How Does it Develop CYK question
"All of the following are ways that children's abilities to solve addition and subtractions problems grow except" is confusing. I'm not sure what the question is asking. Maybe rephrase to something like "All of the following are ways that children can show growth in solving addition and subtraction problems except -" I still think this is a tricky question to answer.

**Comment by ryandkiley** [comment]
TYK Question 1 - The correct answer is "false" but the site incorrectly labels the correct answer as "true"

**Comment by ryandkiley** [comment]
Step 5 "Supporting Students Along the Developmental Trajectory" doesn't come up after clicking "next lesson" in Your Turn. It skips to Adding and Executive Function.

**Comment by castl350** [comment]
Fixed TYK answer - others except video addressed
